sympy/functions/special/delta_functions.py

```diff
@@ -499,8 +539,13 @@ def _eval_rewrite_as_sign(self, arg):
         sign
 
         """
+        if len(args) == 2:
+            arg, H0 = args[0], args[1]
+        else:
+            arg, H0 = args[0], None
         if arg.is_real:
-            return (sign(arg)+1)/2
+            if H0 == None or H0 == S.Half:
+                return (sign(arg)+1)/2
```

Comment:
Ok that makes sense.  Text looks good, maybe: `...must specify Heaviside(0) **as** 1/2...`

"For convenience, we also allow..."?

(Ok, I'll stop bikeshedding).